### DISCUSSION TITLE
New docs colours are harder to read

### DISCUSSION THREAD

**Discussion by DeNeutoy** [fastapi/fastapi#7222] [Questions] [answered]
Hi @tiangolo,

This may be a personal opinion, but I find the new dark grey background for the docs quite difficult to read. The contrast with the sub menu in the top right is smaller, and the code snippets and examples stand out less - just generally I have found myself slower navigating the docs than previously.

Of course you might have had the complete opposite view from others, in which case feel free to close this issue! I thought i'd raise it just in case this background colour change was either accidental or not considered. 

Thanks, and love the lib as always :)

**Comment by DeNeutoy** [comment]
Ah, I think my question is succinctly summarized by this comment: https://github.com/tiangolo/fastapi/issues/725#issuecomment-662447763

although the changes already merged in to address the code highlighting specifically don't change my request.

A manual switch or something would be great!

**Comment by squidfunk** [comment]
Feel free to open an issue at [squidfunk/mkdocs-material](https://github.com/squidfunk/mkdocs-material), which is what FastAPI uses for its docs, ideally with some screenshots explaining the issue.

UPDATE:
https://github.com/squidfunk/mkdocs-material/issues/1862 tracks the problem and should be resolved within the next days.

**Comment by hzjsea** [comment]
I encountered the same problem as you. It seems to be an official adaptation to the color theme. You can change your computer theme to bright colors.

**Comment by squidfunk** [comment]
Material for MkDocs 5.5.7 was just released, which ensures a contrast ratio of 4.5:1 for both, light and dark themes.

**Comment by tiangolo** [comment, marked as answer]
Thanks for the suggestion @DeNeutoy ! :nerd_face: 

And thanks for the discussion here everyone. :coffee: 

@squidfunk has been doing an amazing job with Material for MkDocs and the contrast should be better now, apart from the light/dark toggle. :tada: 

(That was actually done quite some time ago but I'm slowly trying to review all the issues and I got here until now :sweat_smile: )

**Comment by DeNeutoy** [comment]
Ya, and the explicit toggle completely removes this problem anyway, as I can just switch without having to change my laptop's defaults. Thanks!
